browser-test/src/admin_predicates.test.ts

```diff
@@ -355,6 +357,18 @@ describe('create and edit predicates', () => {
           operator: 'is less than',
           values: ['10', '20'],
         },
+        {
+          questionName: 'list of longs',
+          scalar: 'number',
+          operator: 'is one of',
```

Comment:
**suggestion:** Consider changing one of these to "is any of"? Since I think that would allow you to select multiple and it would be displayed differently?

Reply:
This specific bug isn't for multi-value questions, but rather for single value questions where the predicate comparison value is using CSV format to offer multiple multiple values.

Reply:
Done and updated the comparison values used in the test to make that more clear.